test/integration/model/create.test.js

```diff
@@ -1203,6 +1203,27 @@ describe(Support.getTestDialectTeaser('Model'), function() {
       });
     });
 
+    it('should use default value if attribute is excluded in fields option', function() {
+      var User = this.sequelize.define('User', {
```

Comment:
This could be written as a unit test (`User.build()`, and check that `get` returns the right value